### DISCUSSION TITLE
Facing issue while training CRNN model

### DISCUSSION THREAD

**Discussion by Shiva-Ai4m** [mindee/doctr#1274] [General]
I was training the crnn vgg16 model and am facing this issue. How can I fix this issue. 
The image sizes are constant (150*50). Can you please explain me with this,@felixT2K? 
![image](https://github.com/mindee/doctr/assets/122434059/526084b9-f9ac-4a5d-8bef-03191d6a9c55)

**Comment by felixdittrich92** [comment]
`--pretrained` on TF is currently only supported with the default vocab of the model which is for crnn_vgg16_bn `legacy_french`

**Reply by Shiva-Ai4m** [reply]
i worked but i am facing thiis issue now 
![image](https://github.com/mindee/doctr/assets/122434059/c43f7f2f-ec8b-4824-a443-a0d7393495f3)
